composable/chain.json

```diff
@@ -312,6 +312,10 @@
         "address": "https://rpc.centauri.bronbro.io:443",
         "provider": "Bro_n_Bro"
       },
+      {
```

Comment:
New entries needed to be added to the bottom of the list please.

Reply:
Fix it